dashboard/app/models/script.rb

```diff
@@ -1913,4 +1915,16 @@ def self.seed_from_json_file(script_name)
   def self.script_json_filepath(script_name)
     "#{script_json_directory}/#{script_name}.script_json"
   end
+
+  def get_pdf_for_script_overview
```

Comment:
super tiny nit: I think it's worth specifying in the name that these methods get you the _url_ for the pdf, not the pdf itself.